Attestation/ek-secret.md

```diff
@@ -0,0 +1,173 @@
+# `Passing a secret to a TPM using only the EK`
+
+This is example code to pass a secret to a system by just knowing the endorsenment key.
+We will be using the current (commit 07a92e9fa75548ea102ce90b3b6182093b3f7a73 or later) master branch of https://github.com/tpm2-software/tpm2-pytss
+
+The terms for the systems are `remote system`, the system we want to pass the secret to and `local system`, the system which has the secret but doesn't have any TPM.
+One assumtion that will be made is that you already have the EK for the remote system on the local system, and trust it.
+
+## Concept
+The concept is that the key being activated by a call to ActivateCredential on the remote system doesn't have to be generated on the TPM, as long as the public and private parts are loaded it will succeed.
```

Comment:
I think it would be best to start by explaining what tools are available, and that requires talking about restricted keys and what not, and that would quickly blow up in size :(, so maybe we could link to the relevant intros if the intro.

I think I'll send you a PR with my suggestions.

Reply:
My idea is that this more of a sub-topic, so after have read your documentation on attestation this is one alternative to do it, so more "so you understand the overall concepts and here is one way of many to do it"

Reply:
That's fair.  And we should link this from the Intro and the Attestation readmes.

Reply:
+1 to Erik's view on this. I think that the people reading this part of the tutorials would be interested in the particular solution proposed. Thus, understanding the concept, motivation and proposed solution. In that case, keeping a short and clear "Concept" explanation as above is great. 

ps: I think the Concept as it is now summarises the idea behind the solution very well